XRTK-Core/Packages/com.xrtk.core/Definitions/Devices/IMixedRealityHandJointService.cs

```diff
@@ -0,0 +1,22 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using UnityEngine;
+using XRTK.Definitions.Utilities;
+using XRTK.Interfaces.Providers.Controllers;
+
+namespace XRTK.Definitions.Devices
+{
+    /// <summary>
+    /// Mixed Reality Toolkit device definition, used to instantiate and manage a specific device / SDK
+    /// </summary>
+    public interface IMixedRealityHandJointService : IMixedRealityControllerDataProvider
+    {
+        /// <summary>
+        /// Get a game object following the hand joint.
+        /// </summary>
+        Transform RequestJointTransform(TrackedHandJoint joint, Handedness handedness);
+
+        bool IsHandTracked(Handedness handedness);
```

Comment:
Wouldn't this be the same event that gets sent if the input source is found / lost?